sdk/monitor/monitor-query/test/public/shared/testShared.ts

```diff
@@ -117,14 +107,17 @@ export function getAppInsightsConnectionString(mochaContext: Pick<Context, "skip
 }
 
 function getRequiredEnvVar(mochaContext: Pick<Context, "skip">, variableName: string): string {
-  if (!env[variableName]) {
+  const envVar = env[variableName];
+
+  if (!envVar) {
     console.log(
       `TODO: live tests skipped until test-resources + data population is set up (missing ${variableName} env var).`
     );
     mochaContext.skip();
+    // NOTE: This function should be throwing
   }
 
-  return env[variableName];
+  return envVar || "";
```

Comment:
Super minor nit: I tend to prefer `??` over `||` (granted, it doesn't make any practical difference here).

```suggestion
  return envVar ?? "";
```

Reply:
Sounds good for me :)